r/configure

```diff
@@ -68,10 +70,10 @@ else
     PKG_CFLAGS="$PKGCONFIG_CFLAGS"
     PKG_LIBS=${PKGCONFIG_LIBS}
   else
-    if [[ "$OSTYPE" == "darwin"* ]]; then
-      if [ "$FORCE_AUTOBREW" != "TRUE" ] && [ "$(command -v brew)" ] && [ "$(brew ls --versions ${PKG_BREW_NAME})" != "" ]; then
+    if [ "$UNAME" = "Darwin" ]; then
+      if [ "$FORCE_AUTOBREW" != "TRUE" ] && [ "`command -v brew`" ] && [ "`brew ls --versions ${PKG_BREW_NAME}`" != "" ]; then
         echo "Using Homebrew ${PKG_BREW_NAME}"
-        BREWDIR=$(brew --prefix)
+        BREWDIR=`brew --prefix`
```

Comment:
Oh, we couldn't use `$(...)` on Solaris...

Reply:
Right, turns out we could not. This worked, though.